### PR TITLE
update makefile with all instruction

### PR DESCRIPTION
**Pull request by anirrudh**
This change addresses #585 - the assumptions are made as follows:

* A conda environment is activated, either created from the environment.yml file or from the Makefile using the setup instruction.
* localhost:8080/examples/ is the default endpoint

I'm also working on an isolated Dockerfile in parallel. Please let me know if this is enough, or If we need to add the ability to create the conda environment from within the `all` instruction as well.

Cc: @marimeireles

### CODE REVIEW

**Comment by marimeireles** [comment]
Hey @anirrudh!
Thanks for picking this up! ✨
Sorry I opened this issue but I remember I was having a discussion with @fpliger so, do you want a conda environment to be created too? Do you remember what you had in mind?
Also I wonder if `make all` is a good name... Maybe `make install example`? I don't know.
But thanks again for the contribution! =)

File: pyscriptjs/Makefile [hunk 1]
```diff
@@ -64,6 +64,13 @@ test:
 	npm run build
 	$(conda_run) pytest -vv $(ARGS) tests/ --log-cli-level=warning
 
+all:
+	make examples
+	npm install
+	npm run build
```
Comment (fpliger) [inline pyscriptjs/Makefile:70]: I think this can be removed since dev will also build (and then watch)

File: pyscriptjs/Makefile [hunk 2]
```diff
@@ -64,6 +64,13 @@ test:
...
+	npm run dev
```
Comment (fpliger) [inline pyscriptjs/Makefile:71]: I think it's better to change to `make dev` just in case we change cmds in the future...

**Review by fpliger — CHANGES_REQUESTED** [review]
Great, thanks for the contribution @anirrudh ! I left some minor comments but I think the main think for address before merging is the name of the cmd, as @marimeireles pointed out.

Maybe `run-examples`?

**Comment by marimeireles** [comment]
Due to inactivity and #689 being merged, closing this one.
Sorry for the long time to review it @anirrudh, thank you very much for the time here!

**Pull request closed without merging**